bmap/src/lib.rs

```diff
@@ -79,3 +78,17 @@ where
 
     Ok(())
 }
+
+pub fn copy_nobmap<I, O>(input: &mut I, output: &mut O) -> Result<(), CopyError>
+where
+    I: Read + SeekForward,
+    O: Write + SeekForward,
```

Comment:
we don't need the seekforward traits for I and O?

Reply:
I think we do and I have to use it, because as is only works for some files. I think maybe is because those have no "holes". I didn't know why that was happening.